web/src/components/table/use-cases/generations.tsx

```diff
@@ -56,7 +60,7 @@ export type GenerationsTableRow = {
   totalCost?: Decimal;
   traceName?: string;
   metadata?: string;
-  scores: Score[];
+  scores?: Record<string, number>;
```

Comment:
Why is this different now?

Reply:
The changes to the scores are required due to the changes in the getAllQuery.ts
traceId, startTime and completionStartTime (timeToFirstToken) did not have the right type definitions according the prisma schema.